src/http/types.ts

```diff
@@ -25,8 +25,36 @@ export type HttpRequestConfig = AxiosRequestConfig & {
     retryCondition: (error: HttpResponseError) => boolean;
   };
   normalizeCustomData?: boolean;
```

Comment:
Should there be a mention in the comments that this holds only true is `normalizeCustomData` is false?